spec/01-unit/26-tracing_spec.lua

```diff
@@ -0,0 +1,238 @@
+require "spec.helpers" -- initializes 'kong' global for tracer
+
+describe("Tracer PDK", function()
+  local ok, err, _
+
+  lazy_setup(function()
+    local kong_global = require "kong.global"
+    _G.kong = kong_global.new()
+    kong_global.init_pdk(kong)
+  end)
+
+  describe("test tracer init", function()
```

Comment:
may need a test to verify `tracer_type` granually controls which code path we hook into. a test for `"all"` and one for invidual ones will be good.

Reply:
Thanks, added tests for each trace_type.